modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -58,13 +58,20 @@ You cannot have 2 indexes with the same name inside the same bucket and scope.
 . Expand *Mappings* and create a new type mapping on a collection:
 .. Click btn:[Add Type Mapping].
 .. In the *Collection* list, select the collection that contains the documents you want to include in your index. 
-.. Select *Only index specified fields*. 
+.. Select the *only index specified fields* checkbox.
+.. Unset the *#default* field
```

Comment:
Sorry, if we're telling them to "clear the default type mapping" further down, I don't think we need this step. It's the same thing, right?